kairon/shared/channels/mail/processor.py

```
@@ -0,0 +1,271 @@
+            self.smtp.quit()
+            self.smtp = None
+
+
+    @staticmethod
+    def validate_smpt_connection(bot):
+        try:
+            mp = MailProcessor(bot)
+            mp.login_smtp()
+            mp.logout_smtp()
+            return True
+        except:
```

Ruff (0.8.0)

Do not use bare `except`

(E722)